code/api/src/seeders/6-crateproducts.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+const config = require('../config/server.json');
+const params = require('../config/params.json');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('crateproducts', [
+      {
+        crateId: 1,
```

Comment:
To clarify, this represents products that exists in a specific crate, yes?

Reply:
Yes! The data in this table connects a crate with a product.